src/pages/OnboardingEmployees/BaseOnboardingEmployees.tsx

```diff
@@ -0,0 +1,109 @@
+import React, {useMemo, useState} from 'react';
+import {View} from 'react-native';
+import {useOnyx} from 'react-native-onyx';
+import Button from '@components/Button';
+import FormHelpMessage from '@components/FormHelpMessage';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import OfflineIndicator from '@components/OfflineIndicator';
+import ScreenWrapper from '@components/ScreenWrapper';
+import SelectionList from '@components/SelectionList';
+import RadioListItem from '@components/SelectionList/RadioListItem';
+import type {ListItem} from '@components/SelectionList/types';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useResponsiveLayout from '@hooks/useResponsiveLayout';
+import useThemeStyles from '@hooks/useThemeStyles';
+import Navigation from '@libs/Navigation/Navigation';
+import * as Welcome from '@userActions/Welcome';
+import * as OnboardingFlow from '@userActions/Welcome/OnboardingFlow';
+import CONST from '@src/CONST';
+import type {OnboardingCompanySizeType} from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import ROUTES from '@src/ROUTES';
+import type {BaseOnboardingEmployeesProps} from './types';
+
+function BaseOnboardingEmployees({shouldUseNativeStyles, route}: BaseOnboardingEmployeesProps) {
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+    const [onboardingCompanySize] = useOnyx(ONYXKEYS.ONBOARDING_COMPANY_SIZE);
+    const [onboardingPurposeSelected] = useOnyx(ONYXKEYS.ONBOARDING_PURPOSE_SELECTED);
+    const {onboardingIsMediumOrLargerScreenWidth, shouldUseNarrowLayout} = useResponsiveLayout();
+    const [selectedCompanySize, setSelectedCompanySize] = useState<OnboardingCompanySizeType | null | undefined>(onboardingCompanySize);
+    const [error, setError] = useState('');
+
+    const companySizeOptions: ListItem[] = useMemo(() => {
+        return Object.values(CONST.ONBOARDING_COMPANY_SIZE).map((companySize): ListItem => {
+            return {
+                text: translate(`onboarding.employees.${companySize}`),
+                keyForList: companySize,
+                isSelected: companySize === selectedCompanySize,
+                leftElement: onboardingIsMediumOrLargerScreenWidth ? <View style={styles.ml3} /> : null,
+                rightElement: onboardingIsMediumOrLargerScreenWidth ? <View style={styles.mr3} /> : null,
```

Comment:
Can we apply margin/padding to the main element instead of these fake spacers? Also currently the checkbox goes beyond the intended area

<img width="545" alt="Screenshot 2024-09-20 at 5 24 57 PM" src="https://github.com/user-attachments/assets/a5eddbfa-9aaf-44ff-a525-e2a2e91676a0">

Reply:
As I mentioned, `SelectionList` doesn't have a prop to add style to `ListItem`. Then if we want to do that, we need to introduce a new prop.

Reply:
@s77rt Updated with a new prop.